### ISSUE TITLE
Consider options for offline install and document 

### ISSUE THREAD

**Issue by spwoodcock** [hotosm/osm-fieldwork#168]
**Is your feature request related to a problem? Please describe.**

There isn't a good way to install or update osm-fieldwork offline currently.

**Describe the solution you'd like**

Allowing updates to an installed version with with `pip install .` would be ideal, but maybe not feasible. 

Creating the Kivy UI would help solve the issue of install, as all deps would be packaged into an apk and installable entirely offline.

Options should be considered and any solutions documented.

**Comment by spwoodcock** [comment]
We could put that command in the Makefile as a temp workaround until I look into this?

**Comment by spwoodcock** [comment]
This will hopefully be solved by #63.
If we have a bundled version of osm-fieldwork in an app, then we shouldn't need to install osm-fieldwork offline.
If this is still necessary, a simple copy into the site-packages dir should work as Rob suggested.

Something like:

```bash
site_packages_dir=$(python -c "import site; print(site.getsitepackages()[0])")
cp -r "$PWD"/osm_fieldwork/* "$site_packages_dir/osm-fieldwork/"
```
